Identity/IdentityFactory.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  IdentityFactory.swift
+//  RealifeTech
+//
+//  Created by Ross Patman on 14/09/2021.
+//  Copyright © 2020 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+enum IdentityFactory {
```

Comment:
Do this enum and its static function need to be public level?

Reply:
Done